module/VuFind/src/VuFind/Auth/Manager.php

```diff
@@ -668,11 +675,12 @@ public function login($request)
         } catch (\Exception $e) {
             // Catch other exceptions, log verbosely, and treat them as technical
             // difficulties
-            error_log(
-                "Exception in " . get_class($this) . "::login: " . $e->getMessage()
+            $this->logError(
+                "Exception in " . get_class($this) . "::login: "
+                . $e->getMessage()
             );
-            error_log($e);
-            throw new AuthException('authentication_error_technical');
+            $this->logError($e);
```

Comment:
Another observation: `$e` is an exception object in this instance, and `logError` expects a string. This appears to be a pre-existing problem, and maybe automatic typecasting makes it work right, but I think it might be better to explicitly say `$e->getMessage()` here.

Reply:
Isn't better `$e->toString()` because it also logs the stack trace? The error message is logged in the line above `$this->logError($e)`.

Reply:
Yes, that is much better, thank you!